Resources/config/schema/seo-1.0.xsd

```diff
@@ -70,12 +70,12 @@
     <xsd:complexType name="sitemap_configuration">
         <xsd:sequence>
             <xsd:element name="template" type="sitemap_template" minOccurs="1" maxOccurs="unbounded" />
+            <xsd:element name="loader" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
+            <xsd:element name="voter" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
+            <xsd:element name="guesser" type="xsd:string" minOccurs="0" maxOccurs="unbounded" />
         </xsd:sequence>
         <xsd:attribute name="default-change-frequency" type="change_frequency" default="always" />
 
-        <xsd:attribute name="guessers" type="xsd:string" />
-        <xsd:attribute name="loaders" type="xsd:string" />
-        <xsd:attribute name="voters" type="xsd:string" />
```

Comment:
because its a arrayNode now. i remove the split on `,` because that
seems like a hack to me.